.template-helpers/variant.md

```diff
@@ -0,0 +1,7 @@
+# Image Variants
+
+The `%%REPO%%` images come in three flavors, each designed for a specific use case.
+
+`<version>`:
```

Comment:
Looking at the rendered version below, I think these will match the rest of the `README` better if we use headers, and if we qualify them with `%%REPO%%` it'll be more clear exactly what we're talking about:
```markdown
# Image Variants

...

## `%%REPO%%:<version>`

...

## `%%REPO%%:onbuild`

...

## `%%REPO%%:slim`

...
```